lib/nostrum/struct/embed.ex

```diff
@@ -95,7 +95,7 @@ defmodule Nostrum.Struct.Embed do
     |> Map.update(:video, %{}, &Video.to_struct(&1))
     |> Map.update(:provider, %{}, &Provider.to_struct(&1))
     |> Map.update(:author, %{}, &Author.to_struct(&1))
```

Comment:
Good point on whether we should store these as structs. I think the answer is that they should be.

I don't think it's worth your time to go and change every instance of this. Soon I'd like to replace these methods with the macro found [here](https://github.com/Kraigie/nostrum/blob/08ea92bc4a35d1824f8a92a949898d5557b585e0/lib/nostrum/util.ex#L23).